### ISSUE TITLE
Don't show "-> end date" for 1-whole-day events

### ISSUE THREAD

**Issue by BengaloBongali** [plusonelabs/calendar-widget#236]
For all events that last one whole day, the widget shows an arrow beneath and the end date which is of course the same as the start date. This is redundant, because the user knows it's a one-day event if there's neither time nor end date. The end date is not shown if I activate fill up events that last more than one day. I think it sould always be that way for events that last one day.

**Comment by yvolk** [comment]
I cannot reproduce this problem. Your help is needed to provide us with your case, which shows this bug.
Please send me ( or simply paste to this conversation) a report, generated via Calendar Widget Settings -> Feedback -> Share event for debugging

**Comment by yvolk** [comment]
We have more details now, thanks to @neilromig:
1. This behavior is only when "Fill all day events" is turned off.
2. We don't need to show the date no matter if Day headers are on or off.

Correct ?!

**Comment by neilromig** [comment]
Yes, since no date is shown for timed events then no date need be shown for all day events either.

**Comment by yvolk** [comment]
Should be fixed in v.3.0.1, see Beta at https://play.google.com/apps/testing/org.andstatus.todoagenda
?!
